test/e2e/Rakefile

```diff
@@ -48,15 +48,34 @@ end
 task :fixture_wallets_template do
   puts "\n  >> Creating #{FIXTURES_FILE}"
 
-  fixture_wallets = { :linux => { :shelley => mnemonic_sentence(24),
-                                 :icarus => mnemonic_sentence(15),
-                                 :byron => mnemonic_sentence(12) },
-                      :macos => { :shelley => mnemonic_sentence(24),
-                                 :icarus => mnemonic_sentence(15),
-                                 :byron => mnemonic_sentence(12) },
-                      :windows => { :shelley => mnemonic_sentence(24),
-                                   :icarus => mnemonic_sentence(15),
-                                   :byron => mnemonic_sentence(12) }
+  fixture_wallets = { :linux => {
+                          :fixture => { :shelley => mnemonic_sentence(24),
```

Comment:
wonder if :fixture and "target could not be reused. They are repeated for :linux, :macos and :windows

Reply:
The idea is that :fixture are wallets on testnet that hold some ADA and assets. During the tests there are some transactions made from :fixture to :target. At the end of the suite all funds are migrated back from :target to :fixture.

Before this change both fixture and target wallets were restored from genesis and target wallets were always different wallets (from randomly generated mnemonics). Now both :fixture and :target are explicitly created and thanks to this we can use cached wallet-db in tests.